jsManual/antora/antora.yml

```diff
@@ -0,0 +1,14 @@
+name: javascript-manual
+title: Neo4j JavaScript Driver Manual
+version: '4.2'
```

Comment:
The version here is 4.2, while the others are 4.2-preview

Reply:
Gah. It was also missing `prerelease: true`